File: sim-android/im_api/src/main/java/com/sy/im/util/FillEmptyUtil.java
package com.sy.im.util;

import com.sy.im.model.Person;

/**
 * @Author：sy
 * @Date：2023/11/21
 */
public class FillEmptyUtil {
    public static Person setEmpty(Person person){

        if (person.getUserId() == null){
            person.setUserId("");
        }
        if (person.getNickname() == null){
            person.setNickname("");
        }
        if (person.getImgUrl() == null){
            person.setImgUrl("");
        }
        if (person.getGender() == null){
            person.setGender("");
        }
        if (person.getRemark() == null){
            person.setRemark("");
        }
        if (person.getSignature() == null){
            person.setSignature("");
        }
        return person;
    }
}

File: sim-android/app/src/main/java/com/sy/im/util/FillEmptyUtil.kt
package com.sy.im.util;

import com.sy.im.model.Person;

/**
 * @Author：sy
 * @Date：2023/11/21
 */
object FillEmptyUtil {
    fun setEmpty(person:Person ):Person{

        if (person.userId == null){
            person.userId = ""
        }
        if (person.nickname == null){
            person.nickname = ""
        }
        if (person.gender == null){
            person.gender = ""
        }
        if (person.imgUrl == null){
            person.imgUrl = ""
        }
        if (person.remark == null){
            person.remark = ""
        }
        if (person.signature == null){
            person.signature = ""
        }
        return person;
    }
}
